### ISSUE TITLE
Simple64 not working on 4th generation Intel CPUs (Haswell)

### ISSUE THREAD

**Issue by RetroGamingX** [simple64/simple64#459]
simple64 not working on 4th generation Intel CPUs (Haswell). When you load any game the emulator closes.

The last version that worked is "simple64-win64-4ec52c2"

- Core i7 4770K
- Geforce GTX 1070
- OS: Windows 10 22H2

Saludos!

**Comment by tomster785** [comment]
I just tested the latest build with my 4790k and GTX 1080. So same generation for both cpu and gpu. Same OS and version also. It works fine for me, so unless the issue is with your specific models, I don't think the issue is with 4th gen Intel CPUs or GTX 10 series.

**Comment by RetroGamingX** [comment]
> I just tested the latest build with my 4790k and GTX 1080. So same generation for both cpu and gpu. Same OS and version also. It works fine for me, so unless the issue is with your specific models, I don't think the issue is with 4th gen Intel CPUs or GTX 10 series.

Could you share your build so I can test?

**Comment by tomster785** [comment]
Yeah sure, I'm just using the latest build on the release page, 9ecd8e8.

**Comment by RetroGamingX** [comment]
> Yeah sure, I'm just using the latest build on the release page, [9ecd8e8](https://github.com/simple64/simple64/commit/9ecd8e862f5334fc882389fdd6133c3f9ae5a2f8).

Thanks, I just tested and I can confirm that with Haswell CPUs (no refresh) the emulator does not work, when loading any rom it closes without reporting any error.

**Comment by loganmc10** [comment]
It probably isn't related to the CPU. The application wouldn't open at all if the CPU isn't compatible. It is likely something to do with the GPU. I would suggest updating your graphics drivers and trying again

**Comment by Triramama** [comment]
what the heck no emulator is picky to what cpu you have, very sussy

**Comment by RetroGamingX** [comment]
I'm almost sure that the problem is on the CPU side and that with Rosalie GUI the same thing happens when opening a game with the LLE plugins selected...

**Comment by loganmc10** [comment]
RMG doesn't have any CPU requirements (it should work with any CPU).

Simple64 uses Parellel-RDP for video. If you experience the same issue in RMG when using Parallel-RDP, then the problem is certainly related to your GPU, not the CPU.
